src/views/PositionPlot/PositionPlotView.tsx

```diff
@@ -22,23 +25,45 @@ type PanelProps = {
     plotType: 'line' | 'scatter'
 }
 
-const margins = {
-    left: 30,
-    right: 20,
-    top: 20,
-    bottom: 50
-}
-
 const panelSpacing = 4
 
-const PositionPlotView: FunctionComponent<Props> = ({data, width, height}) => {
+export const useTimeseriesMargins = (timeseriesLayoutOpts: TimeseriesLayoutOpts | undefined) => {
+    return useMemo(() => {
+        const {hideTimeAxis, hideToolbar} = timeseriesLayoutOpts || {}
+        if (hideToolbar) {
+            return (
+                {
+                    left: 30,
+                    right: 20,
+                    top: 20,
+                    bottom: hideTimeAxis ? 20 : 50
```

Comment:
Noting the difference in bottom margin based on `hideTimeAxis`--where toolbar + axis gets us 50 pixels but toolbar - axis gets us 40. Is this because the `hideToolbar` (or `hideTimeAxis`) parameter is also driving where the label goes, is that right?

Reply:
Yeah the choices of numbers here is not super clear. If the toolbar is hidden, it means we are trying to save space, so I decrease the margins all around.